main/tests/server/src/com/google/refine/browsing/facets/ListFacetTests.java

```diff
@@ -126,8 +128,9 @@ public void testSelectedEmptyChoice() throws IOException {
         Engine engine = new Engine(project);
 
         ListFacetConfig facetConfig = ParsingUtilities.mapper.readValue(jsonConfig, ListFacetConfig.class);
-        Facet facet = facetConfig.apply(project);
+        ListFacet facet = (ListFacet) facetConfig.apply(project);
         facet.computeChoices(project, engine.getAllFilteredRows());
+        facet.getChoices().sort(Comparator.comparing(choice -> String.valueOf(choice.decoratedValue.label)));
```

Comment:
In general, `facet.getChoices()` might return an immutable list: as a user of this class, I wouldn't assume I can sort the results in place. So I think it would be a cleaner to take a copy of the JSON the facet is serialized to, and do the sorting on the JSON list. After that, you'd be able to assert that the resulting JSON is equal to the expected normalized JSON.

Reply:
Made the changes as per the suggestions. Let me know if you have any questions.